packages/realm-web/package.json

```diff
@@ -11,9 +11,9 @@
     "./dist/bundle.es.js": "./dist/bundle.dom.es.js"
   },
   "scripts": {
-    "prepack": "npm run build",
+    "prepack": "npm run bundle",
```

Comment:
It runs just before `pack` (invoked when running `publish`) which ensures the package gets bundled as part of publishing .. but, I can see that we're explicitly invoking `bundle` on CI, so this could probably be deleted all together 👍 https://github.com/realm/realm-js/blob/main/.github/workflows/publish-package-release.yml#L107C28-L107C47
```suggestion
```